webui/react/src/pages/ModelDetails/ModelDetails.settings.ts

```diff
@@ -94,7 +94,7 @@ const config: SettingsConfig<Settings> = {
       type: number,
     },
   },
-  storagePath: 'model-details',
+  storageKey: 'model-details',
```

Comment:
I think for consistency with the rest of our tables, we should do `model-details-${modelId}` here, so that each model has its own table state.